plugins/in_docker_events/docker_events.c

```diff
@@ -153,6 +155,39 @@ static int in_de_collect(struct flb_input_instance *ins,
             }
         }
     }
+    else if (ret == 0) {
```

Comment:
Could we also add the case for ret == 25?
When using the fluentbit inside a container, the error will be 25 instead of 0:
```
[2021/05/03 17:42:17] [error] [input:docker_events:docker_events.0] read returned error: 25, Inappropriate ioctl for device
```
I have updated the issue description https://github.com/fluent/fluent-bit/issues/3439 to include this error and how to reproduce: